content/client-server-api/modules/voip_events.md

```diff
@@ -171,18 +171,34 @@ In response to an incoming invite, a client may do one of several things:
 
 ##### Streams
 
-Clients are expected to send one stream with one track of kind `audio` (creating a
-voice call). They can optionally send a second track in the same stream of kind
-`video` (creating a video call).
-
-Clients implementing this specification use the first stream and will ignore
-any streamless tracks. Note that in the JavaScript WebRTC API, this means
-`addTrack()` must be passed two parameters: a track and a stream, not just a
-track, and in a video call the stream must be the same for both audio and video
-track.
-
-A client may send other streams and tracks but the behaviour of the other party
-with respect to presenting such streams and tracks is undefined.
+Clients may send more than one stream in a VoIP call. Metadata can be included in
+the [`m.call.invite`](/client-server-api/#mcallinvite), [`m.call.answer`](/client-server-api/#mcallanswer)
+and [`m.call.negotiate`](/client-server-api/#mcallnegotiate) events to describe
+the streams being sent, using the `sdp_stream_metadata` property.
+
+`sdp_stream_metadata` maps from the `id` of a stream in the session description, 
+to metadata about that stream.  Currently only one property is defined for the
+metadata. This is `purpose`, which should be a string indicating the purpose of
+the stream. The following `purpose`s are defined:
+
+*  `m.usermedia` - stream that contains the webcam and/or microphone tracks
+*  `m.screenshare` - stream with the screen-sharing tracks
+
+If an incoming stream is not described in `sdp_stream_metadata` and
+`sdp_stream_metadata` is present, the stream should be ignored. If a stream has
+a `purpose` of an unknown type (i.e. not `m.usermedia` or `m.screenshare`), it
+should be ignored.
+
+Clients implementing this specification will ignore any streamless tracks. Note
```

Comment:
@richvdh [said](https://github.com/matrix-org/matrix-spec/pull/1602/files#r1287281183):

> I know this is existing text, but it now seems even more confusing than before, so I think we need to clarify.
>
> 1. What is the mechanism by which streamless tracks are ignored? Or is this an instruction to clients to ignore such tracks?
> 2. what is that to do with `addTrack`'s parameters?
> 3. is it related to `sdp_stream_metadata`? if not, it should probably not be in the middle of the text about it?

Reply:
1. Indeed. Can you change `will` to `MUST` (or, maybe more readably, `should`)?
2. Right. Yes. I think we should remove it.
3. Yes, I think it would make more sense at the end of the section, after the note on backwards-compatibility.

Reply:
Done